app/request/model.js

```diff
@@ -0,0 +1,11 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class RequestModel extends Model {
+  @attr('string') type;
+  @attr('string') machine;
+  @attr('string') title;
+  @attr('string') downtime;
+  @attr('string') createdDate;
+  @attr('string') status;
+  @belongsTo('user') owner;
```

Comment:
`autoSave` is recommend by author to use, resolve problems with relationships in two directions. When will you create another `model` with `belongsTo` relationship remember about that. Read more here: https://github.com/funkensturm/ember-local-storage
```suggestion
  @belongsTo('user', { autoSave: true }) owner;
```